### PR TITLE
catch all solana error into network down

### PR DESCRIPTION
**Pull request by henrily-ledger**
### 📝 Description

This is a super naive mode on catch error from Solana node that are too frequent and poluate sentry logs with network error

### ❓ Context

- **Impacted projects**: `live-common` 
- **Linked resource(s)**: `` 

### ✅ Checklist

- [ ] **Test coverage** 
- [ ] **Atomic delivery** 
- [ ] **No breaking changes** 

### 🚀 Expectations to reach

_Please make sure you follow these [**Important Steps**](https://github.com/LedgerHQ/ledger-live/blob/develop/CONTRIBUTING.md#important-steps)._

_Pull Requests must pass the CI and be internally validated in order to be merged._

### CODE REVIEW

File: libs/ledger-live-common/src/families/solana/api/chain/index.ts [hunk 1]
```diff
@@ -82,6 +83,11 @@ export type ChainAPI = Readonly<{
   config: Config;
 }>;
 
+// Naive mode, allow us to filter in sentry all this error comming from Sol RPC node
+const remapErrors = (e) => {
+  throw new NetworkDown(e);
```
Comment (haammar-ledger) [inline libs/ledger-live-common/src/families/solana/api/chain/index.ts:88]: Maybe `NetworkError` would be more relevant? (we created it recently)

**Review by haammar-ledger — COMMENTED** [review]
LGTM, just need to run integration tests + bot for non-reg.

File: libs/ledger-live-common/src/families/solana/api/chain/index.ts (hunk 1 above)
Comment (henrily-ledger) [reply libs/ledger-live-common/src/families/solana/api/chain/index.ts:88]: I don't see NetworkError on code base. Can you provide me the file ?

Comment (gre) [reply libs/ledger-live-common/src/families/solana/api/chain/index.ts:88]: also `new NetworkDown(e)` won't make the error correctly mapped, it probably would be better to `new NetworkError(e.message)`

if we introduce such new NetworkError, don't forget to add to the exclusion list of sentry too ( https://ledgerhq.atlassian.net/wiki/spaces/WALLETCO/pages/3757539598/How+to+ignore+an+error+to+be+sent+to+Sentry )

**Review by haammar-ledger — APPROVED** [review]

**Pull request merged**